tests/test_db.py

```diff
@@ -99,6 +99,10 @@
 PERIOD_OUT_OF_BOUNDS_ERROR = "'{type}' date {date} is out of bounds"
 MOVE_ERROR = "identity '0001' is already assigned to 'AAAA'"
 INDIVIDUAL_LOCKED_ERROR = "Individual {mk} is locked"
+INTERVAL_INVALID_ERROR = "'interval' must be a positive number or 0."
+FORMAT_NOT_FOUND_ERROR = "{} not found in the registry"
+FORMAT_EMPTY_ERROR = "'{}' cannot be .+"
+FORMAT_ALREADY_EXISTS = "'{}' already exists in the registry"
```

Comment:
I created those errors because I was declaring the same string multiple times changing only one word. I use those errors later for example: `FORMAT_EMPTY_ERROR.format('backend')`